### PR TITLE
Adapted to new helloworld version based on custom build Golang image.

### PR DESCRIPTION
**Pull request by luebken**
Documentation for https://github.com/giantswarm/helloworld/pull/6

### CODE REVIEW

**Comment by luebken** [comment]
RFR @puja108 @kordless @marians

File: content/guides/annotated-helloworld.md [hunk 1]
```diff
@@ -67,27 +67,24 @@ In case you want to understand what your application is doing, let's have a look
...
+The file configures an application called `helloworld` with a single service. This service contains a single component named `helloworld-component`. This component uses a [custom helloworld image](https://registry.hub.docker.com/u/giantswarm/helloworld/) from the public Docker registry. The image is a simple HTTP server written in Go returning a HTML file. If you are interested in the implementation details have a look at the source code on our [Github page](https://github.com/giantswarm/helloworld). The `ports` definition exposes port 8080 of the component, which is finally exposed via the `domains` entry via standard HTTP port 80.
```
Comment (marians) [inline content/guides/annotated-helloworld.md]: I'd prefer "returning an HTML file"

**Comment by marians** [comment]
LGTM

File: content/guides/annotated-helloworld.md (hunk 1 above)
Comment (puja108) [inline content/guides/annotated-helloworld.md]: True, it should be "an H"

**Pull request merged**